### ISSUE TITLE
email.charset.SHORTEST is not in the stub

### ISSUE THREAD

**Issue by jolaf** [python/typeshed#2830]
```python
from email.charset import SHORTEST
```
Causes `mypy` to complain:
`Module 'email.charset' has no attribute 'SHORTEST'`

However:
```
$ python
>>> from email.charset import SHORTEST
>>> SHORTEST
3
```

**Comment by jolaf** [comment]
Probably the cause is a bug in Python documentation:
https://bugs.python.org/issue33449

**Comment by srittau** [comment]
This affects only the Python 2 stubs. Patches welcome!

**Comment by jolaf** [comment]
Nope, I've actually encountered the problem with Python 3.

**Comment by srittau** [comment]
It was fixed in #2768 for Python 3, but the Python 2 version still needs this fix.

**Comment by jolaf** [comment]
https://github.com/python/typeshed/pull/2854
